devices/sonoff/snzb-03p_motion_sensor.json

```diff
@@ -0,0 +1,192 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "eWeLink",
+  "modelid": "SNZB-03P",
+  "vendor": "SONOFF",
+  "product": "SNZB-03P",
+  "sleeper": false,
```

Comment:
Sure? 🙂

Reply:
Changed something

Reply:
Sorry, my bad. I meant `"sleeper": false,`, as I consider this as rather unusual for a battery powered device.

Reply:
Arf, right, have see the device, no power, and on the GUI it's a end device, no reason for the device was not a slepper